### ISSUE TITLE
Logstash is failing on 7.0.0 BC1

### ISSUE THREAD

**Issue by bhavyarm** [elastic/logstash#10656]
- Version: 7.0.0 BC1
- Operating System: windows but same on ubuntu

I tried to run our integration test on BC1 of 7.0.0. Logstash is failing to come up and these are the logs:
```
[2019-04-05T06:50:43,549][INFO ][logstash.setting.writabledirectory] Creating directory {:setting=>"path.queue", :path=>"C:/windowsInstalls/7.0.0/logstash/data/queue"}
[2019-04-05T06:50:43,591][INFO ][logstash.setting.writabledirectory] Creating directory {:setting=>"path.dead_letter_queue", :path=>"C:/windowsInstalls/7.0.0/logstash/data/dead_letter_queue"}
[2019-04-05T06:50:43,605][FATAL][logstash.runner          ] An unexpected error occurred! {:error=>#<ArgumentError: The setting `xpack.management.elasticsearch.url` has been deprecated and removed from Logstash>, :backtrace=>["C:/windowsInstalls/7.0.0/logstash/logstash-core/lib/logstash/settings.rb:605:in `value'", "C:/windowsInstalls/7.0.0/logstash/logstash-core/lib/logstash/settings.rb:229:in `validate_value'", "C:/windowsInstalls/7.0.0/logstash/logstash-core/lib/logstash/settings.rb:140:in `block in validate_all'", "org/jruby/RubyHash.java:1419:in `each'", "C:/windowsInstalls/7.0.0/logstash/logstash-core/lib/logstash/settings.rb:139:in `validate_all'", "C:/windowsInstalls/7.0.0/logstash/logstash-core/lib/logstash/runner.rb:278:in `execute'", "C:/windowsInstalls/7.0.0/logstash/vendor/bundle/jruby/2.5.0/gems/clamp-0.6.5/lib/clamp/command.rb:67:in `run'", "C:/windowsInstalls/7.0.0/logstash/logstash-core/lib/logstash/runner.rb:237:in `run'", "C:/windowsInstalls/7.0.0/logstash/vendor/bundle/jruby/2.5.0/gems/clamp-0.6.5/lib/clamp/command.rb:132:in `run'", "C:\\windowsInstalls\\7.0.0\\logstash\\lib\\bootstrap\\environment.rb:73:in `<main>'"]}
```
[logstash-plain.log.txt](https://github.com/elastic/logstash/files/3046711/logstash-plain.log.txt)

**Comment by jsvd** [comment]
This is due to https://github.com/elastic/logstash/pull/10623 that was merged between rc2 and bc1
It prevents logstash from starting altogether so it is a major blocker.

**Comment by jsvd** [comment]
fixed in #10657
